src/os/ui/layer/vectorstylecontrols.js

```diff
@@ -80,6 +82,10 @@ os.ui.layer.VectorStyleControlsCtrl = function($scope) {
     this.scope['showCenterIcon'] = true;
   }
 
+  this.scope['lineDashOptions'] = os.style.LINE_STYLE_OPTIONS;
+  var name = os.style.dashPatternToOptions(this.scope['lineDash']);
+  this.scope['lineDashName'] = name ? name : this.scope['lineDashOptions'][0];
```

Comment:
These are very confusing without type declarations. It seems like we have 3 different variables with different names that are all of type `os.style.styleLineDashOption`...?